### PR TITLE
Move jarjar build & publish over to pants.

### PR DESCRIPTION
**Pull request by jsirois**
This kills the ant and maven setups as well as checked in jars and sets
up the repo for maven-central publishing as well.

### CODE REVIEW

**Comment by jsirois** [comment]
Assigning to @ericzundel cc @gmalmquist

**Comment by jsirois** [comment]
I'd like to do the publish of Garrett's jarjar exit code fix in #5 via pants itself.

**Comment by stuhood** [comment]
Any reason not to just import this to pantsbuild/pants?

**Comment by jsirois** [comment]
Probably not - how about baby steps?  I can publish Garretts fix in #5 then move.

That said - this was a good exercise in how hard is it to use pants - which we miss rolling things into pants.

**Comment by jsirois** [comment]
FWIW - I patch these in like so `curl -L https://github.com/pantsbuild/jarjar/pull/6.patch | git am`

File: BUILD.tools [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# JVM artifacts that Pants itself depends on.
+# http://pantsbuild.github.io/dev_tasks.html#jvm-tool-bootstrapping
+
+# common rev for all org.scala-lang%* artifacts
+SCALA_REV = '2.10.5'
```
Comment (ericzundel) [inline BUILD.tools:5]: do we really need this? no scala code here.

Comment (jsirois) [inline BUILD.tools:5]: We do - this is the state of pants.  The configuration is minimal.

Comment (jsirois) [inline BUILD.tools:5]: To be more helpful, ScalaPlatform is in the mix when you use the jvm backend and that requires specs.

**Comment by ericzundel** [comment]
Do you need to initialize pushdb to some initial version? we've already published version 1.5

**Comment by ericzundel** [comment]
sorry wrong button

**Comment by jsirois** [comment]
No - thats via `--override=org.pantsbuild#jarjar=1.5.1` when I do the publish.

**Comment by ericzundel** [comment]
Wow, this is really awful (github pr) I agree with Stu and I think we should just pull it into pants proper too like we already did this with zinc.  That leaves jmake out by its lonesome.  I think we are all about to shed it anyway.  But since you are ready to go with this, LGTM

**Comment by jsirois** [comment]
Patched in @ 8a8bfbd3c0d91231df2fae2570beb7167826c46f

**Pull request closed without merging**